### PR TITLE
Add user to organization

### PR DESCRIPTION
**Pull request by garyluu**
Creating another endpoint that makes it easier to add another user.  Making addUserToOrg into an update for a user that already exists

### CODE REVIEW

File: dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganisationResource.java [hunk 1]
```diff
@@ -362,6 +362,19 @@ public Organisation updateOrganisation(@ApiParam(hidden = true) @Auth User user,
         return organisationDAO.findById(id);
     }
 
+    @PUT
+    @Timed
+    @UnitOfWork
+    @Path("/{organisationId}/users/{username}")
+    @ApiOperation(value = "Adds a user role to an organisation.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = OrganisationUser.class)
+    public OrganisationUser addUserToOrgByUsername(@ApiParam(hidden = true) @Auth User user,
```
Comment (agduncan94) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganisationResource.java]: What is the reason for adding by username and not ID? Is the username unique?

Comment (garyluu) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganisationResource.java]: By ID is impossible to use.  Username is unique.

File: dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganisationResource.java [hunk 2]
```diff
@@ -603,7 +614,7 @@ private boolean doesOrganisationExistToUser(Long organisationId, Long userId) {
 
         // Check that you are not applying action on yourself
         if (Objects.equals(user.getId(), userId)) {
-            String msg = "You cannot add yourself to an organisation.";
+            String msg = "You cannot modify yourself in an organisation.";
```
Comment (denis-yuen) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganisationResource.java]: Thought for discussion, should this be true. 
What happens when curators/admins add themselves to organizations currently?

Comment (agduncan94) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganisationResource.java]: Currently they are able to add themselves.

Comment (denis-yuen) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganisationResource.java]: I may be misunderstanding then. 
Is a different check operating for curators/admins?

Comment (agduncan94) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganisationResource.java]: Oh sorry, no one should be able to edit their own organisation membership. At least not yet, if we decide to change that.

**Pull request merged**